tests/test_libzim.py

```diff
@@ -274,3 +275,58 @@ def get_redirect_url(self):
     with File(path) as reader:
         assert reader.get_article(redirect_url).is_redirect
         assert reader.get_article(redirect_url).get_redirect_article().longurl == url
+
+
+@pytest.mark.parametrize(
+    "no_method", [m for m in dir(Article) if m.split("_", 1)[0] in ("get", "is", "should")],
+)
+def test_article_overriding_required(tmpdir, monkeypatch, no_method):
+    """ ensure we raise properly on not-implemented methods of Article """
+
+    class OverridenArticle(Article):
```

Comment:
We can move the `OverridenArticle` outside of the test and have the `no_method` fixture loop over the method of `OverridenArticle` instead of `Article`.

Reply:
Done.